tests/tests_lite/accelerators/test_cpu.py

```diff
@@ -0,0 +1,54 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
A mix of test cases carried over from tests/pytorch/accelerators/test_cpu.py and newly written ones